app/services/user_updater.rb

```diff
@@ -168,6 +168,8 @@ def update_muted_users(usernames)
   end
 
   def update_ignored_users(usernames)
+    return if user.trust_level < 2
```

Comment:
⚠️ I could have use guardian here, but then we will need to invoke it for every Ignored User (`usernames`)

Reply:
You could have a guardian method `can_ignore_users?` which would do something like

``` ruby
def can_ignore_users?
  SiteSetting.ignore_user_enabled? && (is_staff? || user.trust_level >= TrustLevel.levels[:member])
end
```

Reply:
Yes, though, do we want to have yet another method in the guardian for a seemingly check?

Reply:
```suggestion
    return if user.trust_level < TrustLevel.levels[:member]
```